### PR TITLE
Add handling and tests for TextTransform for SearchBar/Editor/Entry

### LINKED ISSUE #2990: Map Controls.InputView.TextTransform
This will not be a Core property. It will remain on Controls.InputView, and Controls.InputView's subclasses (Editor, Entry, SearchBar)  will need to add custom mappings which apply the current TextTransform.

See the Label.cs partial class under HandlerImpl/Label for an example of a custom Controls-level mapping for this.

### PR DESCRIPTION
**Pull request by hartez**
Fixes #2990

### CODE REVIEW

**Review by rmarinho — CHANGES_REQUESTED** [review]
```
  D:\a\_work\1\s\src\Controls\src\Core\HandlerImpl\Editor\Editor.iOS.cs(7,4): error CS0234: The type or namespace name 'MauiTextViewExtensions' does not exist in the namespace 'Microsoft.Maui.Controls.Platform' (are you missing an assembly reference?) [D:\a\_work\1\s\src\Controls\src\Core\Controls.Core.csproj]
  D:\a\_work\1\s\src\Controls\src\Core\HandlerImpl\Entry\Entry.iOS.cs(7,4): error CS0234: The type or namespace name 'MauiTextViewExtensions' does not exist in the namespace 'Microsoft.Maui.Controls.Platform' (are you missing an assembly reference?) [D:\a\_work\1\s\src\Controls\src\Core\Controls.Core.csproj]
```

File: src/Controls/src/Core/Platform/Android/Extensions/ButtonExtensions.cs [hunk 1]
```diff
@@ -13,7 +13,17 @@ public static class ButtonExtensions
 		public static void UpdateText(this MaterialButton platformButton, Button button)
 		{
 			var text = TextTransformUtilites.GetTransformedText(button.Text, button.TextTransform);
-			platformButton.Text = text;
+
+			try
```
Comment (Redth) [inline src/Controls/src/Core/Platform/Android/Extensions/ButtonExtensions.cs:17]: Accidental debug code merge here?

**Review by rmarinho — APPROVED** [review]

File: src/Controls/src/Core/Platform/Android/Extensions/ButtonExtensions.cs (hunk 1 above)
Comment (hartez) [reply src/Controls/src/Core/Platform/Android/Extensions/ButtonExtensions.cs]: Fixed.

**Pull request merged**